megalinter/descriptors/xml.megalinter-descriptor.yml

```diff
@@ -8,15 +8,25 @@ file_extensions:
   - ".xml"
 linters:
   # XML LINT
-  - linter_name: xmllint
+  - class: XmlLintLinter
+    linter_name: xmllint
+    variables:
```

Comment:
Probably add some comments in linter_description so it will apear at the top of the doc (people won't think to read all the variables )

Reply:
Done